pre-processing/primod/driver_coupling/ribamod.py

```diff
@@ -106,6 +106,25 @@ def write_exchanges(
                     save_flows=True,
                 )
                 destination = "river"
+
+                #  check on the bottom elevation and ribasim minimal subgrid level
+                minimum_subgrid_level = (
+                    ribasim_model.basin.subgrid.df.groupby("subgrid_id")
+                    .min()["subgrid_level"]
+                    .to_numpy()
+                )
+                # in active coupling, check subgrid levels versus modflow bottom elevation
+                if isinstance(self, RibaModActiveDriverCoupling):
+                    subgrid_index = mapping.dataframe["subgrid_index"]
+                    bound_index = mapping.dataframe["bound_index"]
+                    bottom_elevation = package["bottom_elevation"].to_numpy()
+                    if np.any(
+                        bottom_elevation[np.isfinite(bottom_elevation)][bound_index]
+                        > minimum_subgrid_level[subgrid_index]
+                    ):
+                        raise ValueError(
+                            f"Ribasim subgrid levels found below the bottom elevation for MODFLOW 6 package: {key}."
```

Comment:
should be: Found bottom elevation above minimum subgrid level of Ribasim, for MODFLOW 6 package: {key}.